packages/opentelemetry-exporter/test/test.js

```diff
@@ -178,7 +178,11 @@ mochaSuiteFn('Instana OpenTelemetry Exporter', function () {
 });
 
 function verifySpans(spans, appControls) {
-  expectExactlyNMatching(spans, 5, [
+  // ENTRY /otel-test
+  // EXIT www.instana.com
+  // 2 x express middleware, 1 x request handler
+  // 1 x tlc connect, 1 x tls connect
+  expectExactlyNMatching(spans, 7, [
```

Comment:
FYI: When updating `@opentelemetry/auto-instrumentations-node` we automatically install more Otel instrumentations. Therefor more spans will be transformed/exported.

[0.27.1 and 0.36.5](https://github.com/open-telemetry/opentelemetry-js-contrib/compare/auto-instrumentations-web-v0.27.1...auto-instrumentations-node-v0.36.5#diff-99a38dfed350c4dacec0cabcacac3232c7da7a010e6ff9ae2848e9d1bce8e427).